src/cljs/rems/application.cljs

```diff
@@ -1035,22 +1034,22 @@
 
 
 (defn- dynamic-actions [app]
```

Comment:
This could be written like so

```
(defn- dynamic-actions [app]
  (let [command->action {:rems.workflow.dynamic/save-draft [save-button]
                         :rems.workflow.dynamic/submit [submit-button]
                         :rems.workflow.dynamic/return [return-action-button]
                         :rems.workflow.dynamic/request-decision [request-decision-action-button]
                         :rems.workflow.dynamic/decide [decide-action-button]
                         :rems.workflow.dynamic/request-comment [request-comment-action-button]
                         :rems.workflow.dynamic/comment [comment-action-button]
                         :rems.workflow.dynamic/approve [approve-reject-action-button]
                         :rems.workflow.dynamic/reject [approve-reject-action-button]
                         :rems.workflow.dynamic/close [close-action-button]}]
    (distinct (map command->action (:possible-commands app)))))
```

Reply:
But then the buttons will be in unpredictable order (the order of commands in the set).

Reply:
`static-actions` has the buttons in predictable order.

Reply:
Yeah, I guess the original is better.